app/services/transaction_service/paypal_events.rb

```diff
@@ -118,8 +118,11 @@ def pending_ext?(pending_reason)
   end
 
   def pending_ext_to_paid(tx)
-    MarketplaceService::Transaction::Command.transition_to(tx[:id], :paid)
+    # Commission charge is synchronous and must complete before we
+    # transition to paid so that we have the full payment info
+    # available at the time we send payment receipts.
     TransactionService::Transaction.charge_commission(tx[:id])
+    MarketplaceService::Transaction::Command.transition_to(tx[:id], :paid)
```

Comment:
This comment was so convincing that I thought the order here was wrong: https://github.com/sharetribe/sharetribe/pull/2620/files#diff-df4b745a9b64b2271a4aad63144569beL105